src/credit-card/credit-card.ts

```diff
@@ -51,5 +51,6 @@ export function encrypt(cardNumber: string|number, cvv: string|number, month: st
   if(!validate(cardNumber, cvv, month, year)){
     return Observable.throw('Credit card details invalid');
   }
-  return creditCardEncrypt(cleanInput(cardNumber), cleanInput(cvv), Number(cleanInput(month)), Number(cleanInput(year)));
+  // allow CVV to be optional if it is null
+  return creditCardEncrypt(cleanInput(cardNumber), cvv === null ? '' : cleanInput(cvv), Number(cleanInput(month)), Number(cleanInput(year)));
```

Comment:
Ya. TSYS will just turn it into a `''` though. I was debating whether to make that internal detail match the input or the output. I'll change it to `null`.